frontend/src/components/SearchBar/SearchBar.css

```diff
@@ -0,0 +1,63 @@
+.findText {

```

Comment:
NP (nitpick): Would it be possible to use the `kebab-case` standard here?

Reply:
a lot of this I copy/pasted from mainSearch, and just realized I didn't need this so it's gone!